src/rfc-8252-http-server.ts

```diff
@@ -272,6 +272,25 @@ export class RFC8252HTTPServer {
     });
   };
 
+  private static async _getAllInterfaces(
```

Comment:
optional suggestions: Feels a bit silly to mark this `private` and then access it as if it weren't private, no? It's not exported from the package anyway, don't be shy to mark this `public` to make it a bit more obvious that it's tested directly

And while you're at it, you might as well pass in `dns.lookup` as an argument instead of mocking it in that case?

Reply:
I was debating with myself whether I want to make it public or if I want to instead call `.listen()` and then inspect the logs to see what interfaces were discovered. I feel a bit iffy about exposing methods publicly only for the sake of testing, but perhaps that's a bit more acceptable in the JS world 😅

Reply:
If that feels iffy, I think you could also just export it as a separate method, it's fairly standalone and not really tied to the rest of the class semantically 🙂